src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -95,6 +97,35 @@ private ApiMethodDocView generateOptionalArrayMethodDoc(MethodTransformerContext
     return docBuilder.build();
   }
 
+  /*
+   * Constructs a list of the optional fields for the method. Adds the page token field, which is
+   * missing from the optional fields list, and rearrange so that page token and page size are last.
```

Comment:
I think I'd really prefer the other languages to be the "odd ones out", where they have to skip the page token. We shouldn't have to contort PHP to get the full list of optional fields.